Core/Spells: add slowing effect to Mage spell Frostbolt

HannibalRoG opened the pull request:
[//]: # (***************************)
[//]: # (** FILL IN THIS TEMPLATE **)
[//]: # (***************************)

**Changes proposed:**

-  Add slowing effect to frostbolt of Mage, taken from @Keader  https://gist.github.com/Keader/02eaf2fd3fa32e7c29f808802a52c111


**Target branch(es):** 3.3.5/master

- [ ] 3.3.5
- [x] master

**Issues addressed:** Closes #  (insert issue tracker number)
Closes #18932

**Tests performed:** (Does it build, tested in-game, etc.)
Compiled and tested in game

**Known issues and TODO list:** (add/remove lines as needed)

HannibalRoG committed:
Add slowing effect to Frostbolt of Mage.

Edreisantafe commented:
From my starter edition account
![wowscrnshot_021017_013614](https://cloud.githubusercontent.com/assets/19369492/22816742/b7f3f794-ef31-11e6-80d8-0ae1e1bd2501.jpg)

Here it look like Blizzard.

HannibalRoG committed:
Fix =

streetrat commented:
should be valid.
from, tooltip:
Launches a bolt of frost at the enemy, causing $228597s1 Frost damage and slowing movement speed by $205708s1% for $205708d.

Keader commented on sql/updates/world/master/RoG_2017_02_09_01_world.sql:
```diff
@@ -0,0 +1,3 @@
+-- Frostbolt
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_mage_frostbolt';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (228597, 'spell_mage_frostbolt');
```
Missing new line

Keader commented on src/server/scripts/Spells/spell_mage.cpp:
```diff
@@ -463,40 +464,39 @@ class spell_mage_focus_magic : public SpellScriptLoader
-       {
-           PrepareSpellScript(spell_mage_frostbolt_SpellScript);
-
-           void RecalculateDamage(SpellEffIndex /*effIndex*/)
```
it's no needed anymore?

Shauren replied:
nope, shatter now only increases critical chance

Keader commented on src/server/scripts/Spells/spell_mage.cpp:
```diff
@@ -463,40 +464,39 @@ class spell_mage_focus_magic : public SpellScriptLoader
-       }
+public:
+    spell_mage_frostbolt() : SpellScriptLoader("spell_mage_frostbolt") { }
+ 
```
white space xd

Commit:
Update and rename RoG_2017_02_09_01_world.sql to 2017_02_99_19092_world.sql

- added missing NewLine at end of file
- renamed file according to TC PR standards

Commit:
Remove whitespace in spell_mage.cpp

- removed unneeded white space
 (single space characters on blank lines)

HannibalRoG commented:
Any comments?

Keader committed:
rename sql
